Woof/WoofTests/ImageNamesTests.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUI
```

Comment:
nit:

I recommend to have the same dir structure for tests as for the main target. It will help to navigate and find tests easily.